### PR TITLE
Reorganized some private variables in Rigidbody.

### PR DESCRIPTION
**Pull request by liangfok**
Moved private member variables declared at the top of RigidBody to be in the private section. Added documentation to these member variables.

---
This change is [Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/2914)

### CODE REVIEW

File: drake/systems/plants/RigidBody.h [hunk 1]
```diff
@@ -324,6 +320,10 @@ class DRAKERBM_EXPORT RigidBody {
 #endif
 
  private:
+  // TODO(tkoolen): It's very ugly, but parent, dofnum, and pitch also exist
```
Comment (liangfok) [inline drake/systems/plants/RigidBody.h]: @tkoolen I placed your name here based on what I observed from git blame. Please let me know if you were not the original author.

Comment (tkoolen) [inline drake/systems/plants/RigidBody.h]: @RussTedrake wrote that comment I believe, but you can leave my name there if you want.

Comment (liangfok) [inline drake/systems/plants/RigidBody.h]: OK thanks. I'll leave it as is.

**Comment by jamiesnape** [comment]
@drake-jenkins-bot retest this please

**Comment by jamiesnape** [comment]
@liangfok Any chance you can avoid merging/rebasing so many branches at once? Jenkins is way over capacity as they are triggering a job each time.

**Comment by liangfok** [comment]
@jamiesnape OK will do. Sorry about that 😄 .

**Comment by sherm1** [comment]
It would be better if we could up the Jenkins capacity, I think -- is that possible?

**Comment by jamiesnape** [comment]
We would be looking at distributed Jenkins master instances and probably another $250k. This is just a huge spike.

**Comment by jwnimmer-tri** [comment]
Yeah.  If the other PR's aren't destined to merge in their current form (because others must merge first), then having the future PRs stay closed might be best.  Then they can be merged without trigger builds, but still allow for review / visibility, as well as one-off builds of `pr/NNN/merge` should the need arise.

**Comment by liangfok** [comment]
OK closing the PRs that are down-stream from the PR that's currently up for review.

**Comment by liangfok** [comment]
+@amcastro-tri for feature review since this touches `RigidBody*`.
+@sammy-tri for platform review please.

@drake-jenkins-bot linux-gcc-experimental-ros please
@drake-jenkins-bot linux-clang-experimental-ros please
@drake-jenkins-bot linux-gcc-experimental-matlab-ros please

Thanks!

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2914#-:-KOLam66JiH8a5b6ySGs:201265519)*

**Comment by sammy-tri** [comment]
:lgtm: (pending Jenkins)

---

Reviewed 1 of 28 files at r5.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2914#-:-KOLcNK02MhgR4Ce0xkb:583018325)*

**Comment by liangfok** [comment]
CI passed. Now waiting on @amcastro-tri for a feature review.

---

Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2914#-:-KOLjnoPANUdxVS91wRH:-1039899889)*

**Comment by amcastro-tri** [comment]
I just paraphrased the documentation on bitmasks. 

---

Review status: all files reviewed at latest revision, 2 unresolved discussions, all commit checks successful.

---

*[drake/systems/plants/RigidBody.h, line 311 \[r5\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2914#-KOPtgt5_l5t6OsLmFV9:-KOPtgt5_l5t6OsLmFVA:bg1d7t6) ([raw file](https://github.com/robotlocomotion/drake/blob/8a30dc179e7969f4e3d52998dc3eebbff775a8d1/drake/systems/plants/RigidBody.h#L311)):*
> ```objc
>   // A bitmask that determines the collision groups that this rigid body is part
>   // of.
>   DrakeCollision::bitmask collision_filter_group;
> ```

Add a sentence like so: "If the i-th bit is set the the rigid body belongs to the i-th group. A rigid body can belong to several groups."

---

*[drake/systems/plants/RigidBody.h, line 315 \[r5\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2914#-KOPxFZ8uPDmXVSnoclG:-KOPxFZ9c-kI_lFfRqWy:b-baa4bi) ([raw file](https://github.com/robotlocomotion/drake/blob/8a30dc179e7969f4e3d52998dc3eebbff775a8d1/drake/systems/plants/RigidBody.h#L315)):*
> ```objc
>   // A bitmask that determines which collision groups should be ignored when
>   // determinine the colliisons associated with this rigid body.
>   DrakeCollision::bitmask collision_filter_ignores;
> ```

Rephrase to: "A bitmask that determines which collision groups this rigid body does not collide with. Thus, if the i-th bit is set this rigid body is not checked for collisions with bodies in the i-th group."

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2914#-:-KOPxo5grEhjdFipAVKJ:b-exgas5)*

**Comment by liangfok** [comment]
I addressed all reviewer comments. I also added trailing underscores to the member variables and moved some implementation code into `.cc` files. PTAL.

---

Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[drake/systems/plants/RigidBody.h, line 311 \[r5\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2914#-KOPtgt5_l5t6OsLmFV9:-KOQ-ubKB2bRx65eJ0xK:b-896fix) ([raw file](https://github.com/robotlocomotion/drake/blob/8a30dc179e7969f4e3d52998dc3eebbff775a8d1/drake/systems/plants/RigidBody.h#L311)):*
<details><summary><i>Previously, amcastro-tri (Alejandro Castro) wrote…</i></summary>
> Add a sentence like so: "If the i-th bit is set the the rigid body belongs to the i-th group. A rigid body can belong to several groups."
</details>
Done.

---

*[drake/systems/plants/RigidBody.h, line 315 \[r5\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2914#-KOPxFZ8uPDmXVSnoclG:-KOQ-vsGU8wy6uTEQgYC:b-896fix) ([raw file](https://github.com/robotlocomotion/drake/blob/8a30dc179e7969f4e3d52998dc3eebbff775a8d1/drake/systems/plants/RigidBody.h#L315)):*
<details><summary><i>Previously, amcastro-tri (Alejandro Castro) wrote…</i></summary>
> Rephrase to: "A bitmask that determines which collision groups this rigid body does not collide with. Thus, if the i-th bit is set this rigid body is not checked for collisions with bodies in the i-th group."
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2914#-:-KOQAVbhjCgph7LFEXw4:b-465b4q)*

**Comment by amcastro-tri** [comment]
:lgtm:

---

Reviewed 1 of 3 files at r6.
Review status: 1 of 2 files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2914#-:-KOQHy2OMEIRbcUe5t0M:bnfp4nl)*

**Pull request merged**